src/main/java/net/pod/peaengine/gameloop/GameLoop.java

```diff
@@ -20,6 +20,19 @@ public static void setLoopType(Class<? extends GameLoopExecutor> type) {
         GameLoop.type = type;
     }
 
+    /**
+     * Gets current running {@link GameLoopExecutor}
+     * @throws NullPointerException if executor is null
+     * @return current running {@link GameLoopExecutor}
+     */
+    public static GameLoopExecutor getExecutor() {
```

Comment:
The getter for the Thread instance itself poses not quite a "security risk", but is not fool-proof. Please make this method return long instead